lib/TAP.pm

```diff
@@ -726,6 +746,11 @@ my class State does TAP::Entry::Handler {
             }
         }
     }
+    multi method handle-entry(TAP::Comment $comment) {
+        with $!last-test {
+            .add-comment: $comment;
```

Comment:
I think this will cause the comments to be double printed in verbose mode, because they exist both independently and as member of the test.

Reply:
It's ok because they'd be printed at different stages. First time they'd be inlined into the test output. Second time they will be reported in the summary. The latter is way more useful especially in normal mode and on huge test sets.